src/datamap.js

```diff
@@ -26,4 +26,48 @@ export const generate = (genesisHash, size, opts = {}) => {
   return dataMap;
 };
 
-export default { generate, genesisHash: Encryption.genesisHash };
+const rawGenerate = (genesisHash, size) => {
+  const keys = _.range(1, size);
+
+  const [dataMap, _hash] = _.reduce(
+    keys,
+    ([dataM, hash], i) => {
+      const [_obfuscatedHash, nextHash] = Encryption.hashChain(hash);
+      dataM[i] = nextHash;
+
+      return [dataM, nextHash];
+    },
+    [{}, genesisHash]
```

Comment:
I'm not positive but I think this line needs to be `[{}, forge.util.hexToBytes(genesisHash)]`

Reply:
I have no idea, this is what it was in the webnode. Should i change it?

Reply:
Yes